charts/firefly/templates/core/statefulset.yaml

```diff
@@ -54,9 +54,7 @@ spec:
           imagePullPolicy: {{ .Values.core.image.pullPolicy }}
           env:
             - name: FIREFLY_NODE_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.name
+              value: {{ include "firefly.nodeName" . }}
```

Comment:
Might be overkill for now since FF can't have multiple pods / nodes, but part of me wishes we could somehow use `nodeName` as a prefix  in conjunction with the downward API... not asking you to try to see if thats possible but just leaving it as a thought since this will come up eventually again.